sdk/identity/Azure.Identity.Broker/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Support Microsoft Broker on macOS.
```

Comment:
Let's add the redirect uri information in the readme.

e.g. https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity-broker/README.md#redirect-uris